eclipse-collections/pom.xml

```diff
@@ -46,6 +46,18 @@
 
     <build>
 
+        <resources>
+            <resource>
+                <directory>target/generated-sources/resources</directory>
+                <excludes>
+                    <exclude>**/*.crc</exclude>
+                </excludes>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+            </resource>
+        </resources>
```

Comment:
Once the output path is corrected, you won't need this resources config. If that turns out to be impossible, you should change the name from "sources" to "resources", meaning `target/generated-resources`

Reply:
I could probably do this and change to target/generated-resources, but I was mirroring the way src exists with src\main\resources and src\main\java.